README.md

```diff
@@ -239,7 +239,9 @@ By passing `-t lcov` you could generate an lcov.info file and pass it to genhtml
 genhtml -o ./target/debug/coverage/ --show-details --highlight --ignore-errors source --legend ./target/debug/lcov.info
 ```
 
-#### Coveralls/Codecov output
+LCOV output should be used when uploading to Codecov, with the `--branch` argument for partial line coverage support.
```

Comment:
I think this is still a little bit unclear, people might think the support is partial.
I'd go with either "with the `--branch` argument for branch coverage support" or "with the `--branch` argument to enable partial-line coverage".